azurerm/resource_arm_loadbalancer.go

```diff
@@ -100,6 +100,11 @@ func resourceArmLoadBalancer() *schema.Resource {
 				},
 			},
 
+			"sku": {
+				Type:     schema.TypeString,
+				Required: true,
```

Comment:
from what I can tell existing Load Balancers are of the Basic SKU; so we can instead default this to `Basic` and make this Optional; like so:

```
Default: string(network.LoadBalancerSkuNameBasic),
Optional: true,
```

^ this will mean that existing users don't need to specify the SKU (or update the configuration) for any existing Load Balancers. We can also extend this to add Validation and allow mixed-case strings (e.g. `Basic` and `basic`) via:

```
ValidateFunc: validation.StringInSlice([]string{
  string(network.LoadBalancerSkuNameBasic),
  string(network.LoadBalancerSkuNameStandard),
}, true),
DiffSupressFunc: ignoreCaseDiffSuppressFunc,
```